test/cli-parser/handler.spec.js

```diff
@@ -39,6 +39,37 @@ describe('CLI parser', () => {
       assert.equal(compiler.lastArgs[1].plugins.length, 0)
     })
 
+    it('should use first non-hypenated option as source folder if no --source is defined', () => {
+      compiler.reset()
```

Comment:
We should put `compiler.reset()` (or even all the reset calls) in a `beforeEach` block.